<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>container</class>
 <widget class="QWidget" name="container">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tintometro</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="tintometro" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1451</width>
     <height>901</height>
    </rect>
   </property>
   <widget class="QWidget" name="bottom_menuwidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>810</y>
      <width>1441</width>
      <height>91</height>
     </rect>
    </property>
    <widget class="QPushButton" name="home_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/home.png</normaloff>../images/home.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="chrome_btn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/chorme.png</normaloff>../images/chorme.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="status_tintometro">
     <property name="geometry">
      <rect>
       <x>1370</x>
       <y>20</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../images/green.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QStackedWidget" name="bottom_tab">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>1181</width>
       <height>80</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="numeric_keyboard">
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>1200</width>
         <height>84</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="one">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>119</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>57</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="two">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>119</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>57</pointsize>
           </font>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="three">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>119</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>57</pointsize>
           </font>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="four">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>119</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>57</pointsize>
           </font>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="five">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>119</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>57</pointsize>
           </font>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="six">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>119</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>57</pointsize>
           </font>
          </property>
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="seven">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>119</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>57</pointsize>
           </font>
          </property>
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="eight">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>119</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>57</pointsize>
           </font>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nine">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>119</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>57</pointsize>
           </font>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="zero">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>119</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>57</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="dot">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>119</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>57</pointsize>
           </font>
          </property>
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="empty"/>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-10</y>
      <width>1451</width>
      <height>811</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabTintometroImage">
     <widget class="QLabel" name="tintometro_image">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1441</width>
        <height>811</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../images/machine.jpg</pixmap>
      </property>
     </widget>
     <widget class="QWidget" name="STEP_2" native="true">
      <property name="geometry">
       <rect>
        <x>479</x>
        <y>349</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="STEP_9" native="true">
      <property name="geometry">
       <rect>
        <x>479</x>
        <y>629</y>
        <width>61</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="STEP_3" native="true">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>350</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="STEP_8" native="true">
      <property name="geometry">
       <rect>
        <x>749</x>
        <y>629</y>
        <width>61</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="STEP_4" native="true">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>349</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="STEP_7" native="true">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>630</y>
        <width>61</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="pre_STEP_2" native="true">
      <property name="geometry">
       <rect>
        <x>389</x>
        <y>349</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="pre_STEP_9" native="true">
      <property name="geometry">
       <rect>
        <x>399</x>
        <y>639</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="pre_STEP_3" native="true">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>350</y>
        <width>91</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="pre_STEP_8" native="true">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>629</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="pre_STEP_4" native="true">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>350</y>
        <width>91</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="pre_STEP_7" native="true">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>630</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="refilll_HEAD_1_STEP_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>140</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Refill</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refilll_HEAD_3_STEP_3">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>140</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Refill</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refilll_HEAD_5_STEP_4">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>140</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Refill</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refilll_HEAD_2_STEP_9">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>470</y>
        <width>112</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Refill</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refilll_HEAD_4_STEP_8">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>471</y>
        <width>112</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Refill</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refilll_HEAD_6_STEP_7">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>471</y>
        <width>112</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Refill</string>
      </property>
     </widget>
     <widget class="QWidget" name="position_input" native="true">
      <property name="geometry">
       <rect>
        <x>109</x>
        <y>329</y>
        <width>171</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="position_output" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>420</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="STEP_1" native="true">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>340</y>
        <width>81</width>
        <height>80</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="STEP_12" native="true">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>430</y>
        <width>61</width>
        <height>80</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="STEP_11" native="true">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>429</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="STEP_10" native="true">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>620</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="STEP_5" native="true">
      <property name="geometry">
       <rect>
        <x>1200</x>
        <y>330</y>
        <width>61</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="STEP_6" native="true">
      <property name="geometry">
       <rect>
        <x>1200</x>
        <y>620</y>
        <width>61</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="view_status_HEAD_1_STEP_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>200</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Visualizza stato testa 1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="view_status_HEAD_3_STEP_3">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>200</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Visualizza stato testa 3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="view_status_HEAD_5_STEP_4">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>200</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Visualizza stato testa 5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="view_status_HEAD_2_STEP_9">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>520</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Visualizza stato testa 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="view_status_HEAD_4_STEP_8">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>520</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Visualizza stato testa 4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="view_status_HEAD_6_STEP_7">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>520</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Visualizza stato testa 6</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="chrome">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>1441</width>
        <height>811</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="chromeLayout"/>
     </widget>
    </widget>
    <widget class="QWidget" name="modalAction">
     <widget class="QScrollArea" name="scrollAreaStatus">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>10</y>
        <width>711</width>
        <height>791</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="listStatus">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>709</width>
         <height>789</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QScrollArea" name="scrollAreaAction">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>661</width>
        <height>791</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="listActions">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>659</width>
         <height>789</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="modalVisual">
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>1441</width>
        <height>821</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
